src/docs/product/releases/usage/sorting-filtering.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Sorting a Filtering
+sidebar_order: 20
+description: "Learn how to sort and filter releases."
+---
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+## Sorting Releases
+
+On Releases list, use "Sort By" dropdown in the top right to sort releases by:
```

Comment:
Removing the location of the dropdown makes the content a bit more future-proof, in case we ever move that UI element. 

```suggestion
On the **Releases** page, you can use the "Sort By" dropdown to sort releases by:
```